### PR TITLE
Issue and PR templates

### PR DESCRIPTION
**Pull request by hamishtaplin**
### Changes

Adds templates for github issues and pull requests
### Testing

No idea, is it even possible without merge?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Changes
+
+### How to test
+
+### Who can test
```
Comment (warrenbailey) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Who can review?

Comment (hamishtaplin) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Isn't that the same thing as "who can ~~review~~ test"?

**Comment by dhilton** [comment]
LGTM - Mergin'

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (warrenbailey) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Not really, review (code review and test), test (just test)?

Merged now so never mind ;-)

**Pull request merged**